dataeng/resources/warehouse-transforms-ci-dbt.sh

```diff
@@ -11,4 +11,17 @@ dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --prof
 dbt seed --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
 
 dbt run $DBT_RUN_OPTIONS $DBT_RUN_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
-dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
\ No newline at end of file
+dbt test $DBT_TEST_OPTIONS $DBT_TEST_EXCLUDE --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET && true
+if [ $? -eq 1 ]
+then
+    if [ "$WITH_RETRY" == "true" ]
+    then
+        pip install -r ../../tools/ci_scripts/requirements.txt
+        NO_OF_TRIES=3
+        python ../../tools/ci_scripts/rerun_flaky_tests.py --project-path . --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/  --profile $DBT_PROFILE  \
+        --target $DBT_TARGET --exclusions "$DBT_TEST_EXCLUDE" --count $NO_OF_TRIES
```

Comment:
I had to change how the rerun_flaky_tests.py script consumes exclusions on the CLI. Rather than pass it the $DBT_TEST_EXCLUDE variable, can you pass it only the actual exclusions.

Reply:
Ok you mean I should write --exclusions "test_name:relationships" instead of using $DBT_TEST_EXCLUDE variable.

Reply:
In-fact I realized we are not using --exclusions "test_name:relationships" in m2m job so I don't need to pass it here.